Automattic/wp-calypso
# Partner Portal: Add license list filters UI

Context:
* Project thread: pbtFPC-Um-p2
* i4 designs: pbtFPC-103-p2

#### Changes proposed in this Pull Request

* Add a filter menu UI to the license list view. This filter has 4 options: "All Active", "Detached", "Attached", and "Revoked". The same navbar also contains a search button.

_Note: this PR only adds the UI. We will still work on its functionality, so the count might feel off, and it will not filter the current licenses_

![image](https://user-images.githubusercontent.com/5714212/107043016-2d292980-67a1-11eb-9ce5-d124e069cae9.png)

#### Testing instructions

* If you do not have a partner key, please contact Infinity for one or you will be unable to view the UI.
* Checkout PR locally, run `yarn && yarn start-jetpack-cloud`.
* Visit http://jetpack.cloud.localhost:3000/partner-portal and select your partner key, if prompted.
* Check the Nav Bar under the title and confirm it matches the design in the screenshot above
* Make sure you can click through each Nav Item (All Active, Detached, Attached, and Revoked) and that the URL changes accordingly
* Click the Search Icon in the NavBar and, after the search input opens, type something and make sure it changes the `?s=` query param in the URL
